compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -2482,10 +2484,12 @@ fn module_structs(
     let module_id = get_module(self_argument)?;
     let module_data = interpreter.elaborator.get_module(module_id);
     let struct_ids = module_data
-        .type_definitions()
+        .definitions()
+        .definitions()
```

Comment:
:thinking:

Reply:
I know... but it's the only way I found to get the definitions as a `Vec`. (I think I don't understand how `ItemScope` yet)